message_ix/core.py

```diff
@@ -9,7 +9,8 @@
 
 class Scenario(ixmp.Scenario):
 
-    def __init__(self, platform, model, scen, version=None, annotation=None, cache=False):
+    def __init__(self, platform, model, scen, version=None, annotation=None,
+                 cache=False, clone=None):
```

Comment:
Is there a particular reason for passing the to-be-cloned-Scenario as a kwarg?

Reply:
Yes, this is necessary to hook everything up with `ixmp.Scenario`. It's messy because there's a strange pattern in `ixmp` which has two `Scenario` classes, one on the module, and one on the `Platform`.